### PR TITLE
Add idol collections

### PR DESCRIPTION
**Pull request by SpaceNerden**
TODO: Change placeholder paper icon

### CODE REVIEW

**Comment by dlee13** [comment]
You used the old base64 textures which are broken. The base64 values should be copied from this repo: https://github.com/ShiroBlank/HoloItems

**Review by dlee13 — CHANGES_REQUESTED** [review]
I've commented on everything that needs to be fixed.

Also, most of jp gen 6 is missing, but the original HoloItems repo didn't have them, so I understand you don't know what to put for those. We can add those later.

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/lazy.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from cgitb import text
```
[the remark this answers is not in the document]
Comment (SpaceNerden) [reply src/main/java/xyz/holocons/mc/holoitemsrevamp/lazy.py:1]: woops

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/misc/AChan.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    private static final String name = "achan";
+    private static final String base64 = "ewogICJ0aW1lc3RhbXAiIDogMTYyMDE4NTgyMzQ3NCwKICAicHJvZmlsZUlkIiA6ICIzMGQ5MGQ1MGQzZjQ0NzZhOTIyMTZmMDM4OGEzYTlkZSIsCiAgInByb2ZpbGVOYW1lIiA6ICJhY2hhbl9VR0EiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYTA0OGQ2ZGFlZWI4MDgyNjlhZmY1MzMzNzI0YWY4Y2Y4NTY1NmI5MjM2MGI0NTM1M2FmZjcyZWNhMmIwYzZkNiIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9";
+
+    public AChan() {
+        super(name, base64);
+    }
+
+    @Override
+    public @NotNull Component getDisplayName() {
+        return Component.text("AChan")
```
[the remark this answers is not in the document]
Comment (SpaceNerden) [reply src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/misc/AChan.java:22]: ~~No one needs to know~~

**Review by SpaceNerden — COMMENTED** [review]
Fixed

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/en1/IRyS.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import net.kyori.adventure.text.format.TextDecoration;
+import org.jetbrains.annotations.NotNull;
+import xyz.holocons.mc.holoitemsrevamp.collection.Idol;
+
+import java.util.List;
+
+public class IRyS extends Idol {
+
+    private static final String name = "irys";
+    private static final String base64 = "ewogICJ0aW1lc3RhbXAiIDogMTYyNjI2NzA0OTc4MywKICAicHJvZmlsZUlkIiA6ICIyMWUzNjdkNzI1Y2Y0ZTNiYjI2OTJjNGEzMDBhNGRlYiIsCiAgInByb2ZpbGVOYW1lIiA6ICJHZXlzZXJNQyIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9lZDZiMzE1NzE0YTZiZTNlOWYzNWE1Y2FlYTQ0NDUxMTRjYmRhNjExMGM1ZTAzOWI0YjM4MmQ0Mjk1ODcwZWQyIgogICAgfQogIH0KfQ==";
```
[the remark this answers is not in the document]
Comment (TraceLosu) [reply src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/en1/IRyS.java:14]: @SpaceNerden I think you forgot this on your base64 fix commit?

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/gen0/AZKi.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import net.kyori.adventure.text.format.TextDecoration;
+import org.jetbrains.annotations.NotNull;
+import xyz.holocons.mc.holoitemsrevamp.collection.Idol;
+
+import java.util.List;
+
+public class AZKi extends Idol {
+
+    private static final String name = "azki";
+    private static final String base64 = "ewogICJ0aW1lc3RhbXAiIDogMTYyMDE4NDMxMTEzNSwKICAicHJvZmlsZUlkIiA6ICIwMzFmZDg3ZTgzYTQ0M2FkYWY2MjcxYTNlNGRjNWIzNCIsCiAgInByb2ZpbGVOYW1lIiA6ICJBWktpXyIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS80NWE2NTRmMzgzMDJkMjQ1ZTU5ZWM1ZjlmNmNiNDY3NDhjODM0MmNiNTUyZDc5NjUzZjExOThhNWZhYTBhNDY4IiwKICAgICAgIm1ldGFkYXRhIiA6IHsKICAgICAgICAibW9kZWwiIDogInNsaW0iCiAgICAgIH0KICAgIH0KICB9Cn0=";
```
[the remark this answers is not in the document]
Comment (TraceLosu) [reply src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/gen0/AZKi.java:14]: @SpaceNerden You forgot this one too.

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/stars1/Arurandeisu.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+import net.kyori.adventure.text.format.TextDecoration;
+import org.jetbrains.annotations.NotNull;
+import xyz.holocons.mc.holoitemsrevamp.collection.Idol;
+
+import java.util.List;
+
+public class Arurandeisu extends Idol {
+
+    private static final String name = "arurandeisu";
+    private static final String base64 = "ewogICJ0aW1lc3RhbXAiIDogMTYyNDg3MzE2Nzk3MSwKICAicHJvZmlsZUlkIiA6ICI2Mzk3ZWE2NTVkNzE0YTZmYmE2OGM0MzE0OTQxODA5OSIsCiAgInByb2ZpbGVOYW1lIiA6ICJBcnVyYW5kZWlzdSIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9mZTFkNDVjODM5ZDE4YjQ5OGJiYzIxZWExZDBmZGNkYzFjYzA3ZmZiZDgzOTNiMDQyNWYxOTY2YmJiOTYxOGIzIgogICAgfQogIH0KfQ==";
```
Comment (TraceLosu) [inline src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/stars1/Arurandeisu.java:14]: Wrong base64

Comment (dlee13) [inline src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/stars1/Arurandeisu.java:14]: Fix this

**Review by TraceLosu — CHANGES_REQUESTED** [review]

**Review by dlee13 — CHANGES_REQUESTED** [review]
Some base64 strings to fix, Roboco's name, and delete lazy.py

**Comment by SpaceNerden** [comment]
Done

File: src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/misc/Achan.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import net.kyori.adventure.text.format.TextDecoration;
+import org.jetbrains.annotations.NotNull;
+import xyz.holocons.mc.holoitemsrevamp.collection.Idol;
+
+import java.util.List;
+
+public class Achan extends Idol {
+
+    private static final String name = "achan";
+    private static final String base64 = "ewogICJ0aW1lc3RhbXAiIDogMTYyMDE4NTgyMzQ3NCwKICAicHJvZmlsZUlkIiA6ICIzMGQ5MGQ1MGQzZjQ0NzZhOTIyMTZmMDM4OGEzYTlkZSIsCiAgInByb2ZpbGVOYW1lIiA6ICJhY2hhbl9VR0EiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYTA0OGQ2ZGFlZWI4MDgyNjlhZmY1MzMzNzI0YWY4Y2Y4NTY1NmI5MjM2MGI0NTM1M2FmZjcyZWNhMmIwYzZkNiIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9";
```
[the remark this answers is not in the document]
Comment (dlee13) [reply src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/misc/Achan.java:14]:
```suggestion
    private static final String base64 = "eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYTA0OGQ2ZGFlZWI4MDgyNjlhZmY1MzMzNzI0YWY4Y2Y4NTY1NmI5MjM2MGI0NTM1M2FmZjcyZWNhMmIwYzZkNiJ9fX0=";
```

**Review by dlee13 — COMMENTED** [review]
The final thing is A-chan's base64, but I will fix that and then merge. Thank you!

**Review by dlee13 — APPROVED** [review]

**Pull request merged**